gslib/commands/du.py

```diff
@@ -177,10 +181,12 @@ def _PrintInfoAboutBucketListingRef(self, bucket_listing_ref):
       num_objs = 1
 
     if not self.summary_only:
-      sys.stdout.write('%(size)-11s  %(url)s%(ending)s' % {
-          'size': size_string,
-          'url': url_str.encode(UTF8),
-          'ending': self.line_ending})
+      if six.PY2:
+        url_str.encode(UTF8)
```

Comment:
I'm guessing that since this isn't being stored in a variable, it's just encoding to verify that the url_str can be represented using `utf8`?  I.e. it will throw a `UnicodeDecodeError` in PY2 if `url_str` was a bytes type and used `latin1` chars?

Reply:
yeah, it was part of testing, I'm making a quick PR to do some cleanups and this will be part of it.